firebase.json

```diff
@@ -44,7 +53,8 @@
     "storage": {
       "port": 9199,
       "host": "127.0.0.1"
-    }
+    },
+    "singleProjectMode": true
```

Comment:
remove this line

Reply:
This was added when project was setup. Didn't add this manually.